events/setup.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import setuptools
+
+
+def read_requirements(file_):
```

Comment:
We probably don't need the additional complexity in this, as we have no lines starting with "-e" or "http" or "https". You could probably do like https://github.com/mozilla/bugbug/blob/8b59321c5f694788606d7a07008238057292a38f/setup.py#L13.